src/cfn.py

```diff
@@ -300,9 +301,10 @@ def _download(path, destination):
 
 @task
 @requires_aws_stack
-def upload_file(stackname, local_path, remote_path, overwrite=False):
-    with stack_conn(stackname):
-        print('stack:', stackname)
+def upload_file(stackname, local_path, remote_path=None, overwrite=False, node=1):
+    remote_path = remote_path or os.path.join("/tmp", os.path.basename(local_path))
+    with stack_conn(stackname, node=node):
+        print('stack:', stackname, 'node', node)
```

Comment:
maybe we should start using a logger here

Reply:
it's not a debug statement, it's just for the user. if anything it should be on stderr. if we have a logger that doesn't emit any prefix that might be ok, but the benefits of that would be quite tiny.